parts/linux/cloud-init/artifacts/labels.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update Labels for Kubernetes nodes
```

Comment:
nit: s/Update/Ensure/g

Reply:
Are you meaning that all Updates should be replaced with Ensure?

Seems like a reasonable take. The reason for update is that's really what we care about. The ensure if more of a bonus feature since it is a service, but it is doing that more than the updating

Reply:
Might leave both; ```Update/Ensure labels for Kubernetes nodes```

Reply:
just want to a accurate of our logic

Reply:
As it now only runs once on start up when it is updating the labels I think update is the appropriate name.

Changing to: "Updates labels for Kubernetes node"

Reply:
Same as another comment, but I felt like the content should be here:

I could be wrong, but I expect it is possible to run the upgrade/reimage without updating the labels, so ensure could definitely work as the qualifier. I don't have a strong opinion, so am open to changing it if you still think ensure if better